src/views/viewTabs.component.tsx

```diff
@@ -187,7 +200,7 @@ const ViewTabs = () => {
         <TabPanel value={value} label={'Data'}>
           <DataView sessionId={loadedSessionId} />
         </TabPanel>
-        <TabPanel value={value} label={'Plots'}>
+        <TabPanel<TabValue> value={value} label={'Plots'}>
```

Comment:
![image](https://github.com/user-attachments/assets/161f3d75-909c-4161-b7d4-3dcdaaa64b12)
![image](https://github.com/user-attachments/assets/74d031e4-5cc1-4a94-969f-d4e5b64f900e)

`<TabValue>` is needed as it doesn't infer it from the value

Reply:
Interesting, if I remove it then it still infers properly - and for example it's not on the `Data` tabpanel in the current commit so the type-checking must work on the CLI as the CI job is passing. If you want to check that the string literals are the correct type, you could do `'my-string' satisfies TabValue`. It's do we prefer adding the generic to the component, ensuring we use `satisfies` on all string literals or just being happy with `string` type being inferred and not checking against `TabValue`.

Also, I think there's a rogue `TabPanel` in `plotSettingsController.component.tsx` that can also use the new imports :)

Reply:
```
        <TabPanel value={value} label={value}>
          {value === 'Data' && <DataView sessionId={loadedSessionId} />}
          {value === 'Plots' && <PlotList />}
        </TabPanel>
```

if i refactor the code to look like this the type checking work correctly